packages/config/config/devices/0x010f/templates/fgwpx-1x1.json

```diff
@@ -0,0 +1,489 @@
+				{
+					"label": "Disables energy reporting",
+					"value": 0
+				}
+			]
+		},
+		"13": {
+			"label": "Periodic Power Report",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32400,
+			"defaultValue": 3600,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disables reporting",
+					"value": 0
+				}
+			]
+		},
+		"14": {
+			"label": "Periodic Energy Report",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32400,
+			"defaultValue": 3600,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disables reporting",
+					"value": 0
+				}
+			]
+		},
+		"15": {
+			"label": "Self Consumption",
+			"description": "Measuring energy consumed by the Wall Plug itself",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"21": {
+			"label": "Up Value",
+			"description": "Upper power threshold",
+			"valueSize": 2,
+			"unit": "0.1W",
+			"minValue": 100,
+			"maxValue": 18000,
+			"defaultValue": 500,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"22": {
+			"label": "Down value",
+			"description": "Lower power threshold",
+			"valueSize": 2,
+			"unit": "0.1W",
+			"minValue": 0,
+			"maxValue": 17900,
+			"defaultValue": 300,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"23[0x01]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "Power Exceeded controlling 3rd association group devices"
+		},
+		"23[0x02]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "Power Dropped controlling 3rd association group devices"
+		},
+		"24": {
+			"label": "Switch On",
+			"description": "Command frame sent to devices in 2nd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"25": {
+			"label": "Switch Off",
+			"description": "Command frame sent to devices in 2nd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"26": {
+			"label": "Threshold Up",
+			"description": "Command frame sent to the devices in 3rd group",
```

Comment:
Don't understand this either - I think we need to give these labels a good look on your cleanup pass. Some of them are terrible.